src/main/java/org/jabref/gui/maintable/MainTable.java

```diff
@@ -123,6 +134,38 @@ public MainTable(MainTableDataModel model, JabRefFrame frame,
         database.getDatabase().registerListener(this);
     }
 
+    /**
+     * This is called, if a user starts typing some characters into the keyboard with focus on main table.
+     * The tableview will scroll to the cell with the same starting column value and typed string
+     * @param sortedColumn The ListView currently focused
```

Comment:
`@param` and the text should be separated by one empty line.

I think, this is done by the intelliJ auto formatter

Reply:
Yeah, The empty line is necessary. I did not notice that before